server/hotrod/src/main/java/org/infinispan/server/hotrod/HotRodHeader.java

```diff
@@ -53,4 +53,8 @@ public String toString() {
             ", topologyId=" + topologyId +
             '}';
    }
+
+   public boolean isMultimap() {
```

Comment:
-1 as per the flag critics above

Reply:
Yes, this is what I say, that I first used the flag to reuse operations but in the end I ended to create each operation, so, despite the fact that I create a context for multimap specifically on the server, os not useful anymore, so I just left it in case you guys see any additional use for the flag